### PR TITLE
Implemented move functionality to move a file from package to target

### PR DESCRIPTION
**Pull request by abhishek359**
New functionality for moving file from a package to a target directory has been implemented

### CODE REVIEW

File: tools/rosbash/rosbash [hunk 1]
```diff
@@ -1015,3 +1026,4 @@ complete -F "_roscomplete_rossrv" "rossrv"
 complete -F "_roscomplete_roscreate_pkg" "roscreate-pkg"
 complete -F "_roscomplete_roswtf" -o filenames "roswtf"
 complete -F "_roscomplete_rosconsole" "rosconsole"
+complete -F "_roscomplete_file" "rosmv"
```
Comment (dirk-thomas) [inline tools/rosbash/rosbash]: Please add a newline at the end of the file.

File: tools/rosbash/rosfish [hunk 2]
```diff
@@ -1039,3 +1052,4 @@ complete -x -c rossrv         -a '(_roscomplete_rossrv)'
 complete -x -c roscreate-pkg  -a '(_roscomplete_roscreate_pkg)'
 complete -x -c roswtf         -a '(_roscomplete_roswtf)'
 complete -x -c rosconsole     -a '(_roscomplete_rosconsole)'
+complete -x -c rosmv          -a '(_roscomplete_file)'
```
Comment (dirk-thomas) [inline tools/rosbash/rosfish]: Please add a newline at the end of the file.

File: tools/rosbash/rosbash (hunk 1 above)
Comment (abhishek359) [reply tools/rosbash/rosbash]: Added new line

File: tools/rosbash/rosfish (hunk 2 above)
Comment (abhishek359) [reply tools/rosbash/rosfish]: Added new line

**Comment by dirk-thomas** [comment]
Thanks for the contribution.

**Pull request merged**